cypress/integration/inventory/actions-menu.spec.js

```diff
@@ -20,4 +21,18 @@ describe('ui-inventory: actions', () => {
         optionsShouldDisabled.forEach(option => Actions.optionIsDisabled(option, true));
       });
   });
+
+  it('C196753 verifies action menu options after searching and selecting result', () => {
+    const optionsShouldEnabled = [
+      Actions.saveUUIDOption,
+      Actions.saveCQLQueryOption,
+      Actions.exportMARCOption,
+      Actions.showSelectedRecordsOption,
+    ];
+
+    cy.do(InventorySearchPanel.effectiveLocationInput.select([InventorySearchPanel.effectiveLocationValues.mainLibrary]))
+      .then(() => InventorySearchPanel.selectAllCheckboxResult())
+      .then(() => Actions.open())
+      .then(() => optionsShouldEnabled.forEach(option => Actions.optionIsDisabled(option, false)));
+  });
```

Comment:
Why not use interactors directly for the interaction/assertion rather than use 'fragments'?
```
cy.do([MultiSelect('Effective Location').select('Main Library'),
       Button('Select All').click(),
       Button('Actions').click()]);
cy.expect([Action('Save UUID').is({ enabled: true }),
           Action('Save CQL Query').is({ enabled: true }),
           Action('Export Marc').is({ enabled: true }),
           Action('Show Selected Records').is({ enabled: true })]);
```

Reply:
I don't use this because names or ids can be changed in the future, so we decided to use fragment for avoid this
I will try to fix this my strange code with many "then" :)